src/commands/Tools/whois.ts

```diff
@@ -1,13 +1,27 @@
-import { SkyraCommand } from '@lib/structures/SkyraCommand';
+import { SkyraCommand, SkyraCommandOptions } from '@lib/structures/SkyraCommand';
+import { GuildMessage } from '@lib/types';
 import { Colors } from '@lib/types/constants/Constants';
 import { LanguageKeys } from '@lib/types/namespaces/LanguageKeys';
+import { ApplyOptions, CreateResolvers } from '@skyra/decorators';
 import { Emojis, Time } from '@utils/constants';
 import { GuildMember, MessageEmbed, Permissions, PermissionString, Role, User } from 'discord.js';
-import { CommandStore, KlasaMessage } from 'klasa';
+import { Language } from 'klasa';
 
 const sortRanks = (x: Role, y: Role) => Number(y.position > x.position) || Number(x.position === y.position) - 1;
 const { FLAGS } = Permissions;
 
+@ApplyOptions<SkyraCommandOptions>({
+	aliases: ['userinfo', 'uinfo'],
+	cooldown: 15,
+	description: (language) => language.get(LanguageKeys.Commands.Tools.WhoisDescription),
+	extendedHelp: (language) => language.get(LanguageKeys.Commands.Tools.WhoisExtended),
+	requiredPermissions: ['EMBED_LINKS'],
+	runIn: ['text'],
+	usage: '(user:username)'
+})
+@CreateResolvers([
+	['username', (arg, possible, message) => (arg ? message.client.arguments.get('username')!.run(arg, possible, message) : message.author)]
+])
```

Comment:
Ok but is there any reason for this? Just make it optional and default `user = message.author` in the run block

Reply:
The behaviour is completely different, `s!whois #somechannel` for example should give an error, not default to the author.